run.go

```diff
@@ -40,6 +40,10 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Name:  "detach, d",
 			Usage: "detach from the container's process",
 		},
+		cli.BoolFlag{
+			Name:  "no-delete",
+			Usage: "don't auto destory the container after it stopped",
```

Comment:
small typo (s/destory/destroy/)☺️

```suggestion
			Usage: "don't auto-destroy the container after it stopped",
```

Or perhaps

```suggestion
			Usage: "do not delete the container after it exits",
```